src/chrome/content/rules/5ch.xml

```diff
@@ -0,0 +1,45 @@
+<ruleset name="5ch.net">
+	<target host="*.5ch.net" />
+
+	<test url="http://5ch.net/" />
```

Comment:
You need `5ch.net` in the list of `target` in order to enforce HTTPS on `5ch.net`.  However, you don't need a test URL for `5ch.net`. This is because `<target host="5ch.net" />` implicitly created a test URL. Travis will signal an error if you add a duplicated tests. Sorry if I have been causing confusions.

```suggestion
	<target host="5ch.net" />
	<target host="*.5ch.net" />
```

Reply:
@cschanaj I see!